src/hooks/useFetchItems.ts

```diff
@@ -49,9 +49,10 @@ const fetchGetItem = async (
 
 export const useGetItem = (parentId: ParentId) => {
     const currentApi = useApi();
+    const isLivetv = parentId === 'livetv';
     return useQuery({
-        queryKey: ['Item', parentId],
-        queryFn: ({ signal }) => fetchGetItem(currentApi, parentId, { signal }),
+        queryKey: ['Item', parentId, !isLivetv],
+        queryFn: ({ signal }) => !isLivetv ? fetchGetItem(currentApi, parentId, { signal }) : null,
         enabled: !!parentId
```

Comment:
Maybe this should use `enabled` should include the live tv check? That seems like it might be cleaner if it works :man_shrugging:

Reply:
it works as expected.